bitnami/kubeapps/README.md

```diff
@@ -103,7 +514,6 @@ To enable ingress integration, please set `ingress.enabled` to `true`
 
 Most likely you will only want to have one hostname that maps to this Kubeapps installation (use the `ingress.hostname` parameter to set the hostname), however, it is possible to have more than one host. To facilitate this, the `ingress.extraHosts` object is an array.
 
-If you plan to serve Kubeapps under a subpath (eg., `example.com/subpath`), you will have to disable the default path by setting `ingress.hostname=""` and the enter the hostname and path in the extraHost array; for instance: `ingress.extraHosts[0].name="example.com"` and `ingress.extraHosts[0].path="/subpath"`
```

Comment:
Did you remove this because the settings changed? It was only added 5 months ago (one of @antgamdia 's first) after testing that this works as requested by users, it'd be a shame to remove it if it still works as documented.

Reply:
Ah, nm - I see you removed it because it's got its own FAQ entry below. Great.

Reply:
Correct! It was duplicated in the README.md